ansible/README.md

```diff
@@ -14,6 +14,8 @@ sudo pip install ansible==2.0.2.0
 
 #### Vagrant users
 Nothing to be done, Ansible is already installed during vagrant provisioning.
+You can skip setup and prereq steps as those have been done by vagrant for you.  
+You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
 #### Mac users
 It is assumed that a VM has been provisioned using **docker-machine**.
```

Comment:
@csantanapr agreed on the mac readme. I can recall there used to be a better doc but somehow it's gone now. I had started on the mac doc in https://github.com/openwhisk/openwhisk/issues/73 will move your input there.